templates/docs/patterns/hero/index.md

```diff
@@ -0,0 +1,102 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Hero | Patterns
+---
+
+A hero is a prominent banner section typically used to quickly capture the user's attention after they land on the page.
+Depending on the size and composition of your content, you can choose from a variety of hero layouts.
+
+The hero is constructed using the following inputs:
+
+| Input                                                      | Description                                                                                                                                                                            |
+| ---------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <code>layout</code> (**required**)                         | One of: <ul><li><code>50/50</code></li><li><code>50/50-full-width-image</code></li><li><code>25/75</code></li><li><code>75/25</code></li><li><code>fallback</code> (default)</li></ul> |
+| <code>title</code>                                         | Hero title text, displayed as H1                                                                                                                                                       |
```

Comment:
Title is required. The minimal hero is just a title.

Reply:
Mixed up the required on those two, will fix